frontend/web/browser.ts

```diff
@@ -5,28 +5,38 @@ import 'ophan-tracker-js';
 
 import Article from './pages/Article';
 
-const { data, cssIDs } = window.guardian.app;
+// kick off the app
+const go = () => {
+    const { data, cssIDs } = window.guardian.app;
 
-if (module.hot) {
-    module.hot.accept();
-}
+    if (module.hot) {
+        module.hot.accept();
+    }
+
+    const container = document.getElementById('app');
 
-const container = document.getElementById('app');
-
-if (container) {
-    /**
-     * TODO: Remove conditional when Emotion's issue is resolved.
-     * We're having to prevent emotion hydrating styles in the browser
-     * in development mode to retain the sourceMap info. As detailed
-     * in the issue raised here https://github.com/emotion-js/emotion/issues/487
-     */
-    if (process.env.NODE_ENV !== 'development') {
-        hydrateCSS(cssIDs);
+    if (container) {
+        /**
+         * TODO: Remove conditional when Emotion's issue is resolved.
+         * We're having to prevent emotion hydrating styles in the browser
+         * in development mode to retain the sourceMap info. As detailed
+         * in the issue raised here https://github.com/emotion-js/emotion/issues/487
```

Comment:
Do we know if https://github.com/emotion-js/emotion/issues/487 is fixed yet or just closed?

Reply:
I think it was just closed for lack of activity. Probably not fixed 😢

Reply:
It's not fixed.